chain/network-primitives/src/types.rs

```diff
@@ -671,26 +669,27 @@ impl KnownPeerStatus {
 pub struct KnownPeerState {
     pub peer_info: PeerInfo,
     pub status: KnownPeerStatus,
-    pub first_seen: u64,
-    pub last_seen: u64,
+    pub first_seen: UnixTime,
+    pub last_seen: UnixTime,
```

Comment:
Actually, we have `Time: BorshSerialize`, so we can just use `Time` for these fields, no?

Reply:
@matklad Yes, that was my intention.

However, `KnownPeerState` derives `Borsh` and is saved to database.
In order to limit the scope of this PR, I decided to do that in the next PR.

I'm fine doing it in the same PR if you prefer.

We also use `u64` in `network-protocol`. I wanted to create a separate PR, where we change the time from `u64` to `Time`.

Otherwise, we will end up be discussing in this PR, whenever to do `network protocol` change or not.

Reply:
@matklad Ok, I remove `UnixTime`.